data/mods/Aftershock/recipes/ammo_recipes.json

```diff
@@ -72,5 +72,29 @@
     "time": "100 m",
     "autolearn": true,
     "using": [ [ "blacksmithing_standard", 16 ], [ "steel_standard", 4 ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "BRISK_EXERCISE",
```

Comment:
This is a change from the vanilla recipe, is there any reason why this handloading is moving up *two* activity levels, making it more tiring than any other handloading recipe?

Reply:
Fallout from my terrible go at rebasing the first time around. I'll get that fixed.